src/vs/workbench/parts/extensions/electron-browser/extensionsViews.ts

```diff
@@ -98,6 +101,16 @@ export class ExtensionsListView extends ViewsViewletPanel {
 			.map(e => e.elements[0])
 			.filter(e => !!e)
 			.on(this.pin, this, this.disposables);
+
+		this.listActionBar = append(this.extensionsList, $('.list-actionbar-container'));
+		const actionbar = new ActionBar(this.listActionBar, {
+			animated: false
+		});
+		actionbar.addListener(EventType.RUN, ({ error }) => error && this.messageService.show(Severity.Error, error));
+		const installAllAction = this.instantiationService.createInstance(InstallWorkspaceRecommendedExtensionsAction, InstallWorkspaceRecommendedExtensionsAction.ID, localize('installAll', "Install All"));
+		actionbar.push([installAllAction], { icon: true, label: true });
```

Comment:
@ramya-rao-a Not sure if I like this button here. IMO it does not belong here. We generally place such actions in the title bar, overflow menu in the title.

Reply:
Agreed, am working on putting this in the title bar using an icon just like how we have the "Add new file/folder" icons in the file explorer view